Additions/UIView-KIFAdditions.m

```diff
@@ -485,6 +485,8 @@ - (void)dragPointsAlongPaths:(NSArray *)arrayOfPaths {
             }
             UIEvent *eventDown = [self eventWithTouches:[NSArray arrayWithArray:touches]];
             [[UIApplication sharedApplication] sendEvent:eventDown];
+            
+            CFRunLoopRunInMode(UIApplicationCurrentRunMode, DRAG_TOUCH_DELAY, false);
```

Comment:
That little delay is needed because the "touch begin" and "touch moved" events  are to close to each other. If you would like to activate or deactivate the *UIPanGestureRecognizer* in method ```gestureRecognizerShouldBegin:``` depending on it`s velocity value, it can be 0 again.

Actually that modification isn`t tested, but it seams more logical.